src/renderer/services/file-management-system/index.ts

```diff
@@ -447,25 +442,31 @@ export default class FileManagementSystem {
         // If FSS is still available to continue receiving chunks of this upload
         // simply continue sending the chunks
         let lastChunkNumber = fssStatus.chunkStatuses.findIndex(
-          (status) => status !== ChunkStatus.COMPLETE
+          (status) => status !== UploadStatus.COMPLETE
```

Comment:
Is this change correct?  They likely have the same value, but ChunkStatus seems like the right place to store it.

Reply:
If they are always the same enum values then it makes sense to me to combine them

Reply:
Yeah i agree.  I just think the name of the enum shouldn't be so specific to Upload then.